src/pathplanning.org

```diff
@@ -0,0 +1,130 @@
+#+TITLE: Path Planning
+#+AUTHOR: Josh Ting
+#+EMAIL: josh.ting@gatech.edu
+#+REVEAL_THEME: black
+#+REVEAL_TRANS: linear
+#+REVEAL_SPEED: fast
+#+REVEAL_PLUGINS: (notes pdf)
+#+REVEAL_HLEVEL: 1
+#+OPTIONS: toc:nil timestamp:nil reveal_control:t num:nil reveal_history:t tags:nil author:nil
```

Comment:
(you can remove `author:nil` to add your name in the title slide (and the docs) if you want :P